### PR TITLE
added ability to set autoscaling settings

### PR DESCRIPTION
**Pull request by thecodeassassin**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -22,12 +22,9 @@ provider "symbiosis" {
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
-### Required
-
-- `api_key` (String, Sensitive) The ApiKey used to authenticate requests towards Symbiosis.
-
 ### Optional
 
+- `api_key` (String, Sensitive) The ApiKey used to authenticate requests towards Symbiosis.
```
Comment (Styren) [inline docs/index.md:27]: Is this correct?

**Review by Styren — APPROVED** [review]
Just one question

File: docs/index.md (hunk 1 above)
Comment (thecodeassassin) [reply docs/index.md:27]: wdym? that it's optional?

It's not set as optional though. And this is all auto-generated from tfplugindocs

```
		Schema: map[string]*schema.Schema{
			"api_key": {
				Type:        schema.TypeString,
				Required:    true,
				Sensitive:   true,
				DefaultFunc: schema.EnvDefaultFunc("SYMBIOSIS_API_KEY", nil),
				Description: "The ApiKey used to authenticate requests towards Symbiosis.",
			},
			"endpoint": {
				Type:        schema.TypeString,
				Optional:    true,
				DefaultFunc: schema.EnvDefaultFunc("SYMBIOSIS_ENDPOINT", "https://api.symbiosis.host"),
				Description: "Endpoint for reaching the symbiosis API. Used for debugging or when accessed through a proxy.",
			},
		},
		```

Comment (thecodeassassin) [reply docs/index.md:27]: See: https://github.com/hashicorp/terraform-plugin-docs/issues/42

**Pull request merged**